crates/holochain_state/src/chain_lock.rs

```diff
@@ -17,7 +17,7 @@ pub fn is_chain_locked(txn: &Transaction, lock: &[u8]) -> StateMutationResult<bo
             ",
             named_params! {
                 ":lock": lock,
-                ":now": holochain_types::timestamp::now().0
+                ":now": holochain_types::timestamp::now().secs()
```

Comment:
millis vs seconds? not sure if the db table is expecting millis or seconds?
```suggestion
                ":now": holochain_types::timestamp::now()
```

Reply:
This one looks like it's the secs anyway because that's what ToSql does for timestamp under the hood. I like it more explicitly like this.